frontend/src/pages/Calendars/Calendars.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from 'react';
+import PageTemplate from '../../templates/PageTemplate/PageTemplate';
+import CalendarList from '../../components/CalendarList/CalendarList';
+import { constants } from '../../assets/configs/constants';
+
+const Calendars = () => {
+   const [ calendars, setCalendars ] = useState( [] );
+   const [ isLoading, setIsLoading ] = useState( true );
+   useEffect( () => {
+      const controller = new AbortController();
+      const { signal } = controller;
+      
+      fetch( `${constants.url.API_URL}calendars`,{ signal } )
+         .then( ( response ) => response.json() )
+         .then( ( data ) => {
+            const calendarList = data.filter(
+               ( calendar ) => calendar.summary.slice( 0, 5 ) === 'ROOM_',
+            );
+            setCalendars( calendarList );
+            setIsLoading( false );
+         } );
```

Comment:
What if fetching data fails? There should be a `catch` function to handle promise reject. Without it I assume you'd have loading text infinitely without giving a user any info that data isn't actually loading but failed to load. Same for `Room` file below.